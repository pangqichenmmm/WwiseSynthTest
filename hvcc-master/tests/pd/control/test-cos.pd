#N canvas 633 240 351 248 10;
#X obj 32 30 loadbang;
#X obj 32 196 print;
#X obj 101 30 bng 15 250 50 0 empty empty empty 17 7 0 10 -262144 -1
-1;
#X msg 163 112 -2 3 5;
#X msg 67 112 -14.3;
#X msg 32 112 3.7;
#X msg 125 112 0;
#X obj 32 67 t b b b b;
#X obj 32 161 cos;
#X connect 0 0 7 0;
#X connect 2 0 7 0;
#X connect 3 0 8 0;
#X connect 4 0 8 0;
#X connect 5 0 8 0;
#X connect 6 0 8 0;
#X connect 7 0 5 0;
#X connect 7 1 4 0;
#X connect 7 2 6 0;
#X connect 7 3 3 0;
#X connect 8 0 1 0;
