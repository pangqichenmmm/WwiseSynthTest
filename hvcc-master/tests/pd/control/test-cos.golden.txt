[@ 0.000] print: -0.416147
[@ 0.000] print: 1
[@ 0.000] print: -0.162115
[@ 0.000] print: -0.8481
